lib/pbench/server/auth/auth.py

```diff
@@ -85,24 +155,25 @@ def get_auth_token(self, logger):
 
     @staticmethod
     @token_auth.verify_token
-    def verify_auth(auth_token):
+    def verify_auth(auth_token: str) -> Union["User", InternalUser]:
```

Comment:
Technically, I think this is `Optional[Union[User, InternalUser]]`. (And why does `"User"` need to be quoted here?